joomla/joomla-cms
# Update Punycode

Pull Request for Issue #9497
#### Summary of Changes

Update punycode from v1.2.4 to v1.4.1
#### Testing Instructions

This is used for IDN email validation see #2788

Also see the changes made in #3429 which might not be needed anymore as it looks like they were merged into this upstream library https://github.com/bestiejs/punycode.js/issues/20
